test/styleguide/legacy_templates.js

```diff
@@ -1,20 +1,27 @@
+/* global window: false */
 
 // Taken from background.html.
-// Templates are here solely to support the Backbone views rendered in the styleguide.
+// Templates are here solely to support the Backbone views rendered in the Style Guide.
+
+// Note: Any change here must be reflected in background.html to be reflected in the app
+//   and test/index.html to be reflected in the unit tests.
 
 window.Whisper.View.Templates = {
   hasRetry: `
-    {{ messageNotSent }}
-    <span href='#' class='retry'>{{ resend }}</span>
+    {{ messageNotSent }} <span href='#' class='retry'>{{ resend }}</span>
```

Comment:
Oh, one thing you might notice is that since React doesn’t output spaces in the HTML, we might have to tweak certain paddings and margins in places! 😄 See: https://github.com/gasi/Signal-Desktop/pull/1/files#r165859574

Reply:
These aren't rendered with React... yet. Duly noted, however!